package.json

```diff
@@ -4,6 +4,14 @@
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
+    "generateConfigs": "yarn generateGcConfigs && yarn generateMainnetConfigs",
+    "generateGcConfigs": "mustache config/gc.json subgraph.yaml.mustache > subgraph.yaml && yarn codegen",
+    "generateMainnetConfigs": "mustache config/mainnet.json subgraph.yaml.mustache > subgraph.yaml && yarn codegen",
+    "test": "yarn generateConfigs && graph test",
+    "test:env": "node src/scripts/test.js",
+    "test:gc": "yarn generateConfigs && graph test gc",
+    "test:mainnet": "yarn generateConfigs && graph test mainnet",
```

Comment:
One last nitpick: For `test:gc`, run only `generateGcConfigs`. Same thing for `test:mainnet`

Reply:
@alfetopito if I generateMainnetConfig (only) and then run test:mainnet I'm facing this error:

![image](https://user-images.githubusercontent.com/27218623/198288632-9896c418-0353-4907-aff6-943e6a907854.png)

That's why I left it generating both configs.